pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -254,12 +254,12 @@ func NewExporterService(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Service {
 			},
 			Ports: []corev1.ServicePort{
 				{
-					Name:       "8181-tcp",
+					Name:       "export",
 					Port:       8181,
 					TargetPort: intstr.IntOrString{IntVal: 8181},
 				},
 				{
-					Name:       "9091-tcp",
+					Name:       "jmx",
```

Comment:
I think this also needs to be changed to the `jfr-jmx` name.

Reply:
Ah right. I'll correct that, thanks.